### PR TITLE
remove optional webhook since we recommend 1.9

### PR DESCRIPTION
**Pull request by linsun**

### CODE REVIEW

**Comment by linsun** [comment]
also side car injector is already part of istio-demo or istio-auth-demo yaml files

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit f2659e2913f32673365229df53b8dd2d49c65bce

https://app.netlify.com/sites/preliminary-istio/deploys/5b0ec995c9659279d6623434

File: content/docs/setup/kubernetes/quick-start.md [hunk 1]
```diff
@@ -252,9 +252,7 @@ For example, run the following command on a MacOS or Linux system:
 
 ## Installation steps
 
-1.  Install Istio's core components. Choose one of the three _**mutually exclusive**_ options below fo quick install.  However, we recommend you to install
-with the [Helm Chart](/docs/setup/kubernetes/helm-install/) for production installations of Istio to leverage all the options to configure and
-customize Istio to your needs.
+Install Istio's core components. Choose one of the four _**mutually exclusive**_ options below fo quick install.  However, we recommend you to install with the [Helm Chart](/docs/setup/kubernetes/helm-install/) for production installations of Istio to leverage all the options to configure and customize Istio to your needs.
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/quick-start.md]: for quick installation.

File: content/docs/setup/kubernetes/quick-start.md [hunk 2]
```diff
@@ -268,7 +266,7 @@ customize Istio to your needs.
 
     _**OR**_
 
-    b) Install Istio and enforce mutual TLS authentication between sidecars by default. Use this option only on a fresh kubernetes cluster where newly deployed workloads are guaranteed to have Istio sidecars installed.
+    b) Quick Install Istio and enforce mutual TLS authentication between sidecars by default. Use this option only on a fresh kubernetes cluster where newly deployed workloads are guaranteed to have Istio sidecars installed.
```
Comment (ayj) [inline content/docs/setup/kubernetes/quick-start.md]: s/Install/install

**Comment by linsun** [comment]
@ayj @geeknoid thank you!  PTAL

**Pull request merged**